scripts/generate_syncset.py

```diff
@@ -29,31 +32,60 @@ def get_all_yaml_obj(file_paths):
             yaml_objs.append(obj)
     return yaml_objs
 
+def process_yamls(name, directory, obj):
+    o = copy.deepcopy(obj)
+    # Get all yaml files as array of yaml objecys
+    yamls = get_all_yaml_obj(get_all_yaml_files(directory))
+
+    for y in yamls:
+        if 'patch' in y:
+            if not 'patches' in o['spec']:
+                o['spec']['patches'] = []
+            o['spec']['patches'].append(y)
+        else:
```

Comment:
Would we ever have a case when we have both patches and resources?

Reply:
In the case of oauth templates property, if the objects were not over the 250k-ish limit for oc apply I would have put the secret(s) and patch in a single SSS.